lib/events/two_factor_status_change_event.dart

```diff
@@ -1,9 +1,10 @@
-
-
+import 'package:photos/core/configuration.dart';
 import 'package:photos/events/event.dart';
 
 class TwoFactorStatusChangeEvent extends Event {
   final bool status;
 
-  TwoFactorStatusChangeEvent(this.status);
+  TwoFactorStatusChangeEvent(this.status) {
+    Configuration.instance.setTwoFactor(value: status);
```

Comment:
This doesn't seem like right place to do this :). Inside UserService,  when the call to enable or disable 2FA is completed, we can set this flag?

Reply:
Did this in a slightly different way. Attached a listener to to the `TwoFactorStatusChangeEvent` which will set the two factor status flag when the event is fired.